File: corpus/2017q1/_p2q2_ok/_check/A;0332.java

import java.util.Scanner;

public class JavaApplication1 {
    public static void main(String[] args) {

        Scanner teclado = new Scanner (System.in);
        int Cod[] = {300,301,302,303,304,305};
        int qnt=0;
        int cod;
        int choice=0;
        float total=0;
        String[] Item = new String[6];
        double Preco[] = {1.3, 1.4, 1.5, 1.4, 2.2, 1.0};

        Item[0] = "Pão de Queijo";
        Item[1] = "Bauru";
        Item[2] = "Cachorro Quente";
        Item[3] = "Cheeseburger";
        Item[4] = "Suco";
        Item[5] = "Refrigerante";

        ImprimeVetor(Cod, Item, Preco);

        System.out.println();
        System.out.println("Digite o código do produto (300 a 305):");
        cod = teclado.nextInt();        

        while(cod != 0){
            System.out.println("Digite a quantidade do produto:");
            qnt = teclado.nextInt();

            for(int i=0; i < 6; i++){
                if (Cod[i] == cod){
                    choice = i;
                }            
            }

            total += qnt * Preco[choice];

            System.out.println("Digite o código do produto (300 a 305):");
            cod = teclado.nextInt();
        }

        System.out.println("Total da compra: " + total);

    }

    public static void ImprimeVetor(int Vet1[], String Vet2[], double Vet3[]){
            System.out.println("Código" + "Item" + "Preço Unitário");
            for (int i = 0; i <6; i++){
                System.out.print(Vet1[i] + " ");
                System.out.print(Vet2[i] + " ");
                System.out.print(Vet3[i] + " ");
                System.out.println();
            }
    }

}

File: corpus/py/A;0332.py
Scanner teclado = new Scanner (System.in)
Cod = [300,301,302,303,304,305]
int qnt=0
int cod
int choice=0
float total=0
Preco = [1.3, 1.4, 1.5, 1.4, 2.2, 1.0]
ImprimeVetor(Cod, Item, Preco)
print()
print("Digite o código do produto (300 a 305):")
cod = int(input())
while cod != 0:
    print("Digite a quantidade do produto:")
    qnt = int(input())
    for i=0 in range(0, 6):
        choice = i
    print("Digite o código do produto (300 a 305):")
    cod = int(input())
print("Total da compra: " + total)
def ImprimeVetor(int Vet1[], String Vet2[], double Vet3[]):
    print("Código" + "Item" + "Preço Unitário")
    for 0 in range(0, 6):
        print(Vet1[i] + " ")
        print(Vet2[i] + " ")
        print(Vet3[i] + " ")
        print()